spec/commands/process_webhook_event_spec.rb

```diff
@@ -26,4 +26,14 @@ def perform(request, event)
       perform(request, event)
     end
   end
+
+  context 'membership event' do
+    let!(:event) { 'membership' }
+    let!(:request) { {} }
+
+    it 'calls process membership event' do
+      expect_any_instance_of(Github::ProcessMembershipEvent).to receive(:perform)
```

Comment:
Mejor `expect(Github::ProcessMembershipEvent).to receive(:for).with(...)`. Es mejor probar todo lo que puedas de la llamada. No es lo mismo que se llame a que se llame con lo que esperas que se llame